### PR TITLE
Clean up the JavaScript example

### PR DESCRIPTION
**Pull request by mvano**
This brings the example much closer to current discussions. This does not yet take into account the idea of moving pushRegistrationManager to ServiceWorkerRegistration as I think Peter had some questions about that.

This fixes issue #17.

### CODE REVIEW

**Comment by efullea** [comment]
Bryan (@blsaws) was the author of the example. Probably he wants to have a look.

**Comment by mvano** [comment]
Afaik Bryan actually copied that example from a discussion, possibly with some modifications. Either way it was a rough first cut and both Service Workers and Push have changed since then. I think since he has not commented on this pull request, anyone should feel free to review it.

File: index.html [hunk 1]
```diff
@@ -360,102 +360,29 @@
...
+}
+
+// https://example.com/webapp.js
+navigator.serviceWorker.ready.then(function() {
+  navigator.pushRegistrationManager.register().then(
+    function(registration) {
+      console.log(registration.pushRegistrationId);
+      console.log(registration.pushEndpoint);
+      // We now have the registration details that the application server needs
+      // to start pushing messages. From here we can send them to th  server
```
Comment (beverloo) [inline index.html]: nit: s/th /the/.

File: index.html [hunk 2]
```diff
@@ -360,102 +360,29 @@
...
+      // using an asynchronous XMLHttpRequest.
```
Comment (beverloo) [inline index.html]: Perhaps clarify the comment a little bit to (a) indicate where the application server needs to send the messages to, and (b) prevent making good practices sound like requirements?

"The push registration details needed by the application server to push messages to the push service are now available, and can be send to the application service using, for example, an XMLHttpRequest."

File: index.html [hunk 3]
```diff
@@ -360,102 +360,29 @@
...
+    }, function(e) {
+      console.log(e);
```
Comment (beverloo) [inline index.html]: Since this is an example, please annotate why it would be good practice for a user to log |e| here.

**Comment by mvano** [comment]
Thanks Peter! Please take another look.

File: index.html [hunk 4]
```diff
@@ -343,102 +343,42 @@
...
-    // offer the "local" push server for this registration.
-    sender: {
-     apn: apnSenderId,
-     gcm: gcmSenderId,
-     // ...
+// https://example.com/webapp.js
+navigator.serviceWorker.ready.then(function() {
+  navigator.pushRegistrationManager.register().then(
+    function(registration) {
+      // During development we can log the registration details for easier
```
Comment (beverloo) [inline index.html]: micro nit: you repeat "During development" trice. Perhaps removing the comments on lines 355 and 365 might make sense?

**Comment by beverloo** [comment]
Looks good to me. Thanks Michael!

**Comment by mvano** [comment]
Thanks Peter. I don't feel strongly about those comments so I removed them.

@efullea I think this is ready to merge. Could you take a look? Thanks!

**Pull request merged**